nvchecker_source/README.rst

```diff
@@ -16,3 +16,29 @@ use_max_tag
 This source supports `list options`_ when ``use_max_tag`` is set.
 
 .. _list options: https://github.com/lilydjwg/nvchecker#list-options
+
+ALPM files database
+-------------------
+::
+
+  source = "archfiles"
+
+Search package files in a local ALPM files database. The package does not need to be installed.
+
+archfiles
```

Comment:
Better to name it `pkgname`. Using source name as primary config key is due to historical reasons and can be avoided in newer sources.